### DISCUSSION TITLE
Serverless deploy - Bg jobs

### DISCUSSION THREAD

**Discussion by trompx** [wasp-lang/wasp#1738] [Archived]
Hey,
Can we also deploy wasp open saas on serverless infra?
If yes, how does the background jobs can get triggered (as it would need a pull or event driven system)?

**Comment by Martinsos** [comment]
No, not currently! We are looking into serverless support for the future, but currently it is not an option. If we will have it, it will most likely be done in such way that you will be able to deploy most things serverlessly, but for some things like jobs, if you want to use them, you will also need to have a server deployed alongside (but possibly quite small).
